scio-extra/src/main/scala/com/spotify/scio/extra/voyager/syntax/SCollectionSyntax.scala

```diff
@@ -33,30 +33,13 @@ class VoyagerSCollectionOps(@transient private val self: SCollection[VoyagerUri]
    * Load the Voyager index stored at [[VoyagerUri]] in this
    * [[com.spotify.scio.values.SCollection SCollection]].
    *
-   * @param spaceType
-   *   The measurement for computing distance between entities. One of Euclidean, Cosine or Dot
-   *   (inner product).
-   * @param storageType
-   *   The Storage type of the vectors at rest. One of Float8, Float32 or E4M3.
-   * @param dim
-   *   Number of dimensions in vectors.
    * @return
    *   SideInput[VoyagerReader]
    */
   @experimental
-  def asVoyagerSideInput(
-    spaceType: SpaceType,
-    storageType: StorageDataType,
-    dim: Int
-  ): SideInput[VoyagerReader] = {
```

Comment:
@patrickwmcgee Do you want to keep a voyager v1 compatible API ?

Reply:
Yes, the underlying reader we use unless we start to do the work ourselves, `StringIndex` still requires us to use those parameters for now.

Reply:
Good idea. Will integrate the computation